transport/TransportImpl.cpp

```diff
@@ -2448,9 +2459,13 @@ void TransportImpl::setRtxProbeSource(const uint32_t ssrc, uint32_t* sequenceCou
 
 void TransportImpl::runTick(uint64_t timestamp)
 {
+    if (utils::Time::diffLT(_lastTickJobStartTimestamp, timestamp, utils::Time::ms * 100))
```

Comment:
I would move the check to enginemixer to avoid looping through streams every 10ms.